### ISSUE TITLE
Large tables collapsing or overflowing

### ISSUE THREAD

**Issue by ggenzone** [asciidoctor/asciidoctor-pdf#1668]
Hello, 

We have a problem with the following file

```
= Example 
:pdf-page-size: letter

[cols="8,8,8,20,20,36",width=100%,options="header"]
|===

| Service
| Protocol
| Port
| Source
| Destination
| Purpose

.10+| Directory | TCP | 88 | Node management LIF | Active Directory forest | Kerberos V authentication
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup
| DHCP | UDP | 68 | Node management LIF | DHCP | DHCP client for first-time setup

|===
```

The output is:
[test.pdf](https://github.com/asciidoctor/asciidoctor-pdf/files/4541499/test.pdf)

I don't know the reason but if the header attribute is deleted, the problem disappears.

Regards
guido

**Comment by ggenzone** [comment]
Excuse me, I just saw the error. There is one more column.

**Comment by mojavelinux** [comment]
Yep. And the overlap is addressed by #1623.
